### PR TITLE
use class selector for the toggleContainer also

### PR DESCRIPTION
**Pull request by carolirod**
@notwaldorf I forgot to include the container to be css-selected by a class... even though it doesn't have a mixin applying to the .toggle-container/#toggleContainer, and therefore wouldn't be a problem...

I heard though it's a good practice to style selecting with classes rather than with ids, leaving ids for the interaction with JS etc (told me my senior colleague :) )

thanks!

### CODE REVIEW

File: paper-toggle-button.html [hunk 1]
```diff
@@ -155,7 +155,7 @@
       }
     </style>
 
-    <div id="toggleContainer">
+    <div id="toggleContainer" class="toggle-container">
```
Comment (notwaldorf) [inline paper-toggle-button.html]: I don't think this is used anywhere, other than for this previous styling, so I think you could remove the id altogether :)

**Comment by notwaldorf** [comment]
@carolirod Small comment, otherwise this is good to merge. Thanks!

**Comment by carolirod** [comment]
@notwaldorf sure! there you go, no id now. thanks to you!

**Comment by Naiara80** [comment]
Hi, I've customized the width and height of the .toggle-bar but the container has a fixed height of 14px and the component doesn't take the real height, it would be possible to create a mixin for the .toogle-container?Thanks

**Pull request merged**